src/bootstrap.js

```diff
@@ -2,22 +2,44 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-/* globals XPCOMUtils, UAOverrider UAOverrides */
+/* globals XPCOMUtils, Services, UAOverrider UAOverrides */
 
 Components.utils.import("resource://gre/modules/XPCOMUtils.jsm");
+XPCOMUtils.defineLazyModuleGetter(this, "Services", "resource://gre/modules/Services.jsm");
 XPCOMUtils.defineLazyModuleGetter(this, "UAOverrider", "chrome://webcompat/content/lib/ua_overrider.js");
 XPCOMUtils.defineLazyModuleGetter(this, "UAOverrides", "chrome://webcompat/content/data/ua_overrides.js");
 
 let overrider;
+let tabUpdateHandler;
 
 function install() {} // eslint-disable-line no-unused-vars
 function uninstall() {} // eslint-disable-line no-unused-vars
 
-function startup() { // eslint-disable-line no-unused-vars
+function startup({webExtension}) {
   overrider = new UAOverrider(UAOverrides);
   overrider.init();
+
+  // Init webExtension to listen tab update status
+  webExtension.startup().then(api => {
```

Comment:
Can this ever throw? If so, we should probably add a `catch()` here. (also, that's what our eslint rules suggest)